README.md

```diff
@@ -158,8 +158,39 @@ The CLI offers you a convinient way to create and retrieve data from the PDS. Yo
 ```
 pds-cli
 ```
-
 [The PDS CLI documentation](golang/pds-cli/doc/pds-cli.md) section has detailed instructions of the available options
+## PDS cli demo
+
+Demonstrating the PDS cli you can view the following gifs.
+
+1. Adding a class to a instance
+
+Steps
+* Check the content of site.pp and hiera.yaml to allow for class and configuration on the server
+* Check the content of the client catalog on the client
+* Add the motd class to the client node with the pds-cli on the server
+* Running the puppet agent on the client node and confirm the new class is in the catalog and applied.
+
+![Add a class with PDS](./docs/images/pdsaddfact.gif)
```

Comment:
I love the idea @davidsandilands 👍  great initiative! 

While the terminal/font size is small, it looks good when you Zoom in.

![image](https://user-images.githubusercontent.com/1958982/167171200-b527d4d0-8987-4c20-87be-0f36bb5f41f4.png)

I wish it could go a bit slower though, because it's hard to follow it (lot's of changes in a short period of time), but it's doable if you watch the GIF a couple of times :)

Reply:
Let me have another shot with text size and see if i can get another gif recorder I was limited to 30 seconds